### PR TITLE
Removed iframe/redirect usersync urls in synacormedia config

### LINKED ISSUE #2542: Bidders imds, synacormedia define endpoints for the same syncer key
When I run Prebid Server with the configuration below, I get the following error.

```
jcabalugaz@arcspan prebid-server-go % ./prebid-server
F0125 18:50:01.920778   11653 main.go:53] prebid-server failed: user sync (1 error):
  1: bidders imds, synacormedia define endpoints (iframe and/or redirect) for the same syncer key, but only one bidder is permitted to define endpoints

// pbs.yaml
gdpr:
  default_value: false
```

I believe this issue was introduced when https://github.com/prebid/prebid-server/pull/2508 was merged. It seems that the usersync iframe/redirect keys should be defined in `imds.yaml` or `synacormedia.yaml` but not both.

### PR DESCRIPTION
**Pull request by josecu**
Fixes #2542.

### CODE REVIEW

File: static/bidder-info/synacormedia.yaml [hunk 1]
```diff
@@ -13,10 +13,4 @@ capabilities:
...
 userSync:
   key: "imds"
-  supportCors: true
-  iframe:
-    url: "https://ad-cdn.technoratimedia.com/html/usersync.html?gdpr={{.GDPR}}&consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&cb={{.RedirectURL}}"
-    userMacro: "[USER_ID]"
-  redirect:
-    url: "https://sync.technoratimedia.com/services?srv=cs&gdpr={{.GDPR}}&consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&cb={{.RedirectURL}}"
-    userMacro: "[USER_ID]"
+  supportCors: true
```
Comment (guscarreon) [inline static/bidder-info/synacormedia.yaml:16]: We mignt not need line 16 either. Can we please remove?

**Review by guscarreon — COMMENTED** [review]
@josecu thanks a lot for addressing this issue. Left one comment in my review below.

**Review by bsardo — APPROVED** [review]
LGTM

**Review by mansinahar — APPROVED** [review]

**Comment by SyntaxNode** [comment]
@josecu Thank you for quickly identifying and solving this issue. I take blame for missing this in the original PR. We'll get a new release out shortly.

**Pull request merged**